src/main.cpp

```diff
@@ -1737,7 +1737,6 @@ int main(int argc, char *argv[] )
             material_manager->addSharedMaterial(materials_file);
         }
         Referee::init();
-        powerup_manager->loadAllPowerups();
```

Comment:
I don't think you can remove that. This loads all the models and materials, and this needs to happen at this point in the init sequence. Maybe the parts that loads models and textures, and the parts that load weights, could be separated

Reply:
Okay, that makes sense. I'll separate the weights loading from the model/texture loading.

Reply:
Should be good now. I've retested locally and I could launch and race without issue with those latest changes.

The good thing with this error of mine is that now the splitting allows to not reload the models on each new race.